handler/nats_config.go

```diff
@@ -12,14 +13,12 @@ type NatsConfig interface {
 type DefaultNatsConfig struct {
 }
 
-var supportedCharacters, _ = regexp.Compile("[^a-zA-Z0-9-_]+")
```

Comment:
moved to `nats.GetClientID` so the code is not duplicated in the `handler` package and `main` package

Reply:
@rgee0 yep `MustCompile` seems more appropriate, thanks.